Services/OutputCacheServices.cs

```diff
@@ -0,0 +1,38 @@
+﻿using RestSharp;
+using System.IO;
+
+namespace AndcultureCode.CSharp.Sitefinity.Testing.Services
+{
+    public class OutputCacheService
+    {
+        public string AuthKey { get; set; }
```

Comment:
Does it make sense to add a comment here describing where to set up this auth key in Sitefinity?

Reply:
Done.